src/DateTimeParser.jl

```diff
@@ -17,6 +17,11 @@ export parse, tryparse
 # http://new-pds-rings-2.seti.org/tools/time_formats.html
 # http://search.cpan.org/~muir/Time-modules-2003.0211/lib/Time/ParseDate.pm
 
+immutable DayOfWeek <: DatePeriod
+    value::Int64
+    DayOfWeek(v::Number) = new(v)
+end
+
 const HMS = Dict{AbstractString, Symbol}(
```

Comment:
HMS and AMPM need to allow for locales. Maybe you want to add `"de l’après-midi" => :pm`